azure-pipelines.perf-test.yml

```diff
@@ -21,9 +21,9 @@ jobs:
         displayName: yarn
 
       - script: |
-          NorthstarAffected=$(yarn --silent check:affected-package --@fluentui/react-northstar)
-          V8Affected=$(yarn --silent check:affected-package --@fluentui/react)
-          ReactComponentsAffected=$(yarn --silent check:affected-package --@fluentui/react-components)
+          NorthstarAffected=$(yarn --silent check:affected-package --packages @fluentui/react-northstar --pr)
```

Comment:
``--pr`` variable has the same behavior as the previous usage of the command but it is more explicit now